packages/server/graphql/types/GitLabIntegration.ts

```diff
@@ -34,14 +35,25 @@ const GitLabIntegration = new GraphQLObjectType<any, GQLContext>({
     sharedProviders: {
       description: 'The non-global providers shared with the team or organization',
       type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(IntegrationProviderOAuth2))),
-      resolve: async ({teamId}: {teamId: string}, _args: unknown, {dataLoader}) => {
+      resolve: async (
+        {teamId, userId}: {teamId: string; userId: string},
+        _args: unknown,
+        {dataLoader}
+      ) => {
         const team = await dataLoader.get('teams').loadNonNull(teamId)
         const {orgId} = team
         const orgTeams = await dataLoader.get('teamsByOrgIds').load(orgId)
         const orgTeamIds = orgTeams.map(({id}) => id)
-        return dataLoader
+        const sharedProviders = await dataLoader
           .get('sharedIntegrationProviders')
           .load({service: 'gitlab', orgTeamIds, teamIds: [teamId]})
+        if (!sharedProviders) {
```

Comment:
`sharedProviders` should always be non-null here, no? I thought if there are no results matching the query, it will return an empty array.

Reply:
It should be but that bug in Sentry suggests that `gitlab` can be null which suggests that `sharedProviders` must be null too

Reply:
hmmm... I bet if we will ever see this new error in Sentry